static/app/views/issueDetails/groupActivityItem.tsx

```diff
@@ -551,6 +551,13 @@ function GroupActivityItem({
       case GroupActivityType.SET_ESCALATING: {
         return getEscalatingMessage(activity.data);
       }
+      case GroupActivityType.SET_PRIORITY: {
+        const {data} = activity;
+        return tct(
+          '[actor] updated the priority value of this issue to be X priority after [reason]',
```

Comment:
Should replace `X` here

I also don't know if we want to use the `reason` variable straight up because it might not flow well in a sentence. Probably should return two different strings

```ts
switch (data.reason) {
  case 'escalating':
	return tct('[actor updated the priority value of this issue to be [priority] after it escalated')
  case 'ongoing':
    return tct('... after it was marked as ongoing')
}
```